### DISCUSSION TITLE
RawValue weicht von Recognition-Werten ab

### DISCUSSION THREAD

**Discussion by Pergamon77** [jomjol/AI-on-the-edge-device#2902] [Q&A] [answered]
Hi,
nachdem mein System eine ganze Weile fehlerfrei lief, habe ich seit gestern das Problem, dass die Werte unter Data --> Recognition richtig erkannt werden, aber im Feld rawValue steht ein in einer Stelle abweichender Wert. Dadurch kommt auch eine neg. Rate zustande.
Das Feld Value bleibt ganz leer.
Ich habe schon versucht, den PreValue mit dem aktuellen Wert zu überschreiben, aber es wird danach wieder 140,xxx anstatt 141,xxx angezeigt.
Ich hoffe, durch die Screenshots wird klar, was ich meine.

MaxRateValue ist aktiviert, AllowNegativeRates steht auf false.
![1](https://github.com/jomjol/AI-on-the-edge-device/assets/128689716/588a2bb1-362d-4d9e-befb-52df3bf77ad1)
![2](https://github.com/jomjol/AI-on-the-edge-device/assets/128689716/6c9e60bf-3663-4ea5-9ca4-a941c7e7a9f4)

Danke für die Hilfe!

**Comment by rgiese11** [comment, marked as answer]
Da bist du nicht der einzige. Ich habe dasselbe Problem und bin wieder auf Version 15.4.0 zurück. Nun läuft wieder alles.

**Reply by Pergamon77** [reply]
Achso, danke für den Hinweis!
Ich hatte vorher natürlich nach diesen Fehlern gesucht, aber nur ältere Artikel gefunden, die auch nicht so richtig auf meine Fehler passen.

**Reply by Pergamon77** [reply]
Ich bin jetzt auch wieder bei 15.4.0 und siehe da, es funktioniert!
Damit hatte ich es gar nicht in Verbindung gebracht, da mein letztes Update schon ein paar Tage her ist.
